docs/api-reference/next.config.js/rewrites.md

```diff
@@ -13,6 +13,15 @@ description: Add rewrites to your Next.js app.
   </ul>
 </details>
 
+<details>
```

Comment:
Could you add this to `headers.md` too? Then good to go!

Reply:
I've added it to headers as well 👍